CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * Minimum Gradle version: 6.8.3.
 * Minimum Android Gradle Plugin version: 4.1.3.
 * Minimum Android SDK: 16.
+* Minimum R8: 8.0.27.
```

Comment:
Any reason we are bumping the minimum?

Reply:
Earlier versions of R8 does not know how to read Kotlin 1.9 metadata. This will cause the consumer project to fail because it will not properly honor the configuration when obfuscating out library.  https://developer.android.com/build/kotlin-support